Description: Don't build abc when building yosys.
  It is built as a separate source package.
Forwarded: doesn't make sense upstream
Author: Ruben Undheim <ruben.undheim@gmail.com>
Index: yosys/Makefile
===================================================================
--- yosys.orig/Makefile	2014-10-28 21:25:22.411413201 +0100
+++ yosys/Makefile	2014-10-28 21:27:48.631415935 +0100
@@ -130,7 +130,7 @@
 
 ifeq ($(ENABLE_ABC),1)
 CXXFLAGS += -DYOSYS_ENABLE_ABC
-TARGETS += yosys-abc$(EXE)
+#TARGETS += yosys-abc$(EXE)
 endif
 
 ifeq ($(ENABLE_VERIFIC),1)
